### PR TITLE
refactor: update scExpression docs

### PR DESCRIPTION
**Pull request by pablo-gar**
## Changes

- Docs for scExpression are updated to reflect the latest data processing implementation
- Figures have been updated accordingly, and they use the new color scheme
- Sections have been re-ordered for better readability

### CODE REVIEW

File: frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx [hunk 1]
```diff
@@ -1,28 +1,30 @@
-# scExpression Documentation
+# scExpression — Query Gene Expression Across Tissues
+
+scExpression is a tool that allows to query the expression of any gene across all data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```
Comment (MDunitz) [inline frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx:3]:
```suggestion
scExpression is a tool that allows users to query the expression of any gene across all data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```

Comment (MDunitz) [inline frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx:3]: should we specify that its not actually querying all of the data in the data portal?

Comment (pablo-gar) [reply frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx:3]: you mean because of "is_primary_data"?

Comment (MDunitz) [reply frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx]: that and the assay limits. And we only use datasets with 1 organism (although that only rules out 1 dataset I think)

File: frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx [hunk 2]
```diff
@@ -1,28 +1,30 @@
-# scExpression Documentation
+# scExpression — Query Gene Expression Across Tissues
+
+scExpression is a tool that allows users to query the expression of any gene across all data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```
Comment (pablo-gar) [inline frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx:3]:
```suggestion
scExpression is a tool that allows users to query the expression of any gene across data in the cellxgene data portal. A query results in a dot plot per tissue as explained below.
```

Comment (pablo-gar) [reply frontend/doc-site/04__Analyze Public Data/4_2__scExpression Documentation/4_2_1__Get Started.mdx:3]: @MDunitz would this be better? I don't think I'd want to get into many details in this intro sentence

**Review by MDunitz — APPROVED** [review]
lgtm

**Review by seve — APPROVED** [review]
code looks good

**Pull request merged**